src/NLog.Web.AspNetCore/NLog.Web.AspNetCore.csproj

```diff
@@ -73,6 +77,10 @@ Supported platforms:
     <PackageReference Include="Microsoft.AspNetCore.Hosting.Abstractions" Version="2.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Routing.Abstractions" Version="2.1.1" />
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp3.0' ">
+    <PackageReference Include="Microsoft.Extensions.Http" Version="3.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting.Abstractions" Version="3.0.0" />
```

Comment:
@snakefoot this is a good reason to support the new stuff,  the packages are renamed :|